zerver/tests/test_home.py

```diff
@@ -116,6 +116,8 @@ class HomeTest(ZulipTestCase):
         "password_max_length",
         "presences",
         "presence_last_update_id",
+        # TODO: Only mobile clients need this data actually.
+        "push_accounts_registration_statuses",
```

Comment:
I think the web app may want data along these lines, but merged with data on which devices the user is logged into.